cf_units/__init__.py

```diff
@@ -1883,27 +1887,26 @@ def convert(self, value, other, ctype=FLOAT64):
             32.0
             >>> a64 = np.arange(10, dtype=np.float64)
             >>> c.convert(a64, f)
-            array([ 32. ,  33.8,  35.6,  37.4,  39.2,  41. ,  42.8,  44.6, \
- 46.4,  48.2])
+            array([ 32. ,  33.8,  35.6,  37.4,  39.2,  41. ,  42.8,  44.6,
+                    46.4,  48.2])
```

Comment:
Hmmm, I wonder if this would break the doctest now. Either way, it looks better! :+1: